components/lib/rating/Rating.spec.js

```diff
@@ -37,7 +37,7 @@ describe('Rating.vue', () => {
     it('When star is clicked, onOptionClick method should triggered', async () => {
         await wrapper.find('.p-rating-item').trigger('click');
 
-        expect(wrapper.find('.p-focus').exists()).toBe(true);
+        expect(wrapper.find('[data-p-focused="true"]').exists()).toBe(true);
```

Comment:
`p-focus` is not added on click, just keyboard nav, so looking for the data attribute instead.